roles/docker-htpc/defaults/main.yml

```diff
@@ -4,6 +4,16 @@ downloads_dir:    /mnt/someshare
 vpn_network_cidr: 192.168.1.0/24
 mullvad_account_id: 1234567887654321
 
+# OpenVPN Client doesn't really support Docker tags
+# see https://github.com/dperson/openvpn-client/issues/40
+openvpn_client_version: latest
```

Comment:
See https://github.com/dperson/openvpn-client/issues/40 and https://github.com/dperson/openvpn-client/issues/93